app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/initialgainorloss/InitialGainOrLossController.scala

```diff
@@ -174,7 +197,7 @@ object InitialGainOrLossController {
           }
       }
 
-    val formatter = ConditionalRadioUtils.formatter("initialGainOrLoss")(
+    val formatter = ConditionalRadioUtils.formatter(s"initialGainOrLoss", Some(s"initialGainOrLoss$userKey"))(
```

Comment:
we've handled different user types at the template level for all the other pages - could we stick to that pattern for consistency?